src/SFML/Network/Ftp.cpp

```diff
@@ -620,7 +619,7 @@ void Ftp::DataChannel::send(std::istream& stream)
             break;
         }
 
-        count = static_cast<std::size_t>(stream.gcount());
+        const auto count = static_cast<std::size_t>(stream.gcount());
 
         if (count > 0)
```

Comment:
Do we gain anything be not just inlining this?

I guess we cast the "signed" `streamsize` to unsigned `size_t`, but maybe that's not correct to begin with?

Reply:
I'm not sure I get your concern -- the behavior is equivalent to before, the scope of `count` just used to be unnecessarily large.

Reply:
I'm talking about this
```suggestion
        if (stream.gcount() > 0)
```
or this
```suggestion
        if (static_cast<std::size_t>(stream.gcount()) > 0)
```